code/modules/clothing/under/miscellaneous.dm

```diff
@@ -231,35 +182,6 @@
 	item_state = "r_suit"
 	worn_state = "red_suit"
 
-/obj/item/clothing/under/overalls
-	name = "laborer's overalls"
-	desc = "A set of durable overalls for getting the job done."
-	icon_state = "overalls"
-	item_state = "lb_suit"
-	worn_state = "overalls"
-
-/obj/item/clothing/under/pirate
```

Comment:
Used by antags.

Reply:
Not readding this. We've got better options